<launch>
    <arg name="node_name" default="linear_actuator_controller" />

    <node
        name="$(arg node_name)"
        pkg="pandora_linear_actuator_controller"
        type="pandora_linear_actuator_controller_node"
        respawn="false"
        output="screen" >
        <rosparam command="load"
            file="$(find pandora_linear_actuator_controller)/config/linear_actuator_controller.yaml"/>
    </node>
</launch>
